docs/src/development/development.md

```diff
@@ -0,0 +1,92 @@
+# Developer Guide
+
+## Install prerequisites
+
+1. Install [go][go]
+    - Get the latest patch version for go v1.17.
+1. Install [KIND][kind]
+    - `GO111MODULE="on" go get sigs.k8s.io/kind@v<stableVersion>`.
+1. Install [Kustomize][kustomize]
+    - `brew install kustomize` on macOS.
+    - `choco install kustomize` on Windows.
+    - [install instructions][kustomizelinux] on Linux
+1. Install [envsubst][envsubst]
```

Comment:
Please provide instructions here directly instead of asking the developer to go to another site to look how to install.